packages/io/amf-deserializer/index.js

```diff
@@ -20,25 +18,31 @@ History:
 // See http://amf.wikispaces.com/
 //
 // //////////////////////////////////////////
+
 const translate = require('./translate')
-const deserializeToCSG = require('./deserialize')
+const instantiate = require('./deserialize')
 
 /**
- * Parse the given AMF source (xml) and return either a JSCAD script or a CSG/CAG object
- * @param {string} input amf data
- * @param {string} filename (optional) original filename of AMF source
- * @param {object} options options (optional) anonymous object with:
- * @param {string} [options.version='0.0.0'] version number to add to the metadata
- * @param {boolean} [options.addMetadata=true] toggle injection of metadata (producer, date, source) at the start of the file
- * @param {string} [options.output='jscad'] {String} either jscad or csg to set desired output
- * @return {CSG/string} either a CAG/CSG object or a string (jscad script)
+ * Deserialize the given AMF source (xml) into either a script or an array of geometry
+ * @param {String} input - AMF source data
+ * @param {String} [filename] - original filename of AMF source
```

Comment:
it should stlll indicate that it is optional right ?

Reply:
Corrct. [filename] means optional.